service-front/app/test/ActorTest/Form/CreateNewActivationKeyTest.php

```diff
@@ -32,12 +32,13 @@ public function getFormName(): string
     public function getFormElements(): array
     {
         return [
-            '__csrf'                => Csrf::class,
-            'reference_number'      => Hidden::class,
-            'first_names'           => Hidden::class,
-            'last_name'             => Hidden::class,
-            'dob'                   => Hidden::class,
-            'postcode'              => Hidden::class,
+            '__csrf'                    => Csrf::class,
+            'reference_number'          => Hidden::class,
+            'first_names'               => Hidden::class,
+            'last_name'                 => Hidden::class,
+            'dob'                       => Hidden::class,
+            'postcode'                  => Hidden::class,
+            'force_activation_key'      => Hidden::class,
```

Comment:
```suggestion
            'force_activation_key' => Hidden::class,
```
It's not important for this PR but going forward could we please ensure that this kind of formatting uses a single space for the longest line, all other lines are based off that.